tests/test_channels.py

```diff
@@ -47,3 +49,33 @@ def test_channel_matchspec():
             assert record["channel"] == "conda-forge"
         elif record["name"] == "python":
             assert record["channel"] == "pkgs/main"
+
+
+def test_channels_prefixdata():
+    """
+    Make sure libmamba does not complain about missing channels
+    used in previous commands.
+
+    See https://github.com/conda/conda/issues/11790
+    """
+    with make_temp_env("conda-forge::xz", "python") as prefix:
+        output = check_output(
```

Comment:
Note we need a whole subprocess here because libmamba prints to STDERR directly, and that won't be intercepted with our naïve in-process capturing mechanisms.